### ISSUE TITLE
Expose detach() in .d.ts typings

### ISSUE THREAD

**Issue by FirstVertex** [marcj/css-element-queries#314]
We have a `detach()` method but it is not exposed in the lib's typings.

Let's expose `detach()` so that transient usage of this can be properly disposed.

Currently I have to use casts to unknown and any to get at the detach from Typescript:

```ts
(<any>(<unknown>ElementQueries)).detach();
```
